cypress/cypress.config.js

```diff
@@ -35,30 +36,18 @@ module.exports = {
           global.patientName = name;
```

Comment:
I think there's more in here we can cleanup -- do we want to do that in this PR or in a separate one? really up to you, bob!